wgsl/index.bs

```diff
@@ -9805,6 +9807,13 @@ The rules for analyzing expressions take as argument both the expression itself
       <td class="nowrap">*CF*, *CF*
       <td>
    <tr><td>identifier [=resolves|resolving=] to function-scope variable "x"
+      where the [=load rule=] is not invoked during [=type checking=]
```

Comment:
Yep.
This aligns uniformity analysis with how we do type-checking and the conceptual model of how execution/evaluation occurs.
This is the end result of what @jimblandy suggested, and it works out really nicely.